package.json

```diff
@@ -55,9 +55,9 @@
         "substs": "https://github.com/yunxing/substs.git",
         "easy-format": "https://github.com/npm-opam/easy-format",
         "merlin-extend": "https://github.com/npm-opam/merlin-extend",
-        "opam-installer-bin": "https://github.com/yunxing/opam-installer-bin.git",        
+        "opam-installer-bin": "https://github.com/yunxing/opam-installer-bin.git",
         "nopam": "https://github.com/yunxing/nopam.git",
-        "utop": "https://github.com/npm-opam/utop",
+        "utop-bin": "https://github.com/reasonml/utop-bin",
```

Comment:
Do you want to just name it `utop`? that way people can specify overrides for their top level app to build it from source?

```
"utop": "https://github.com/reasonml/utop-bin"
```